docs/API.md

```diff
@@ -7,47 +7,28 @@ hidden: true
 slug: ue-api
 ---
 
-## Table of content

```

Comment:
The table of content need to stay for the Github users

Reply:
Restored the table